handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -236,6 +236,8 @@
      */
     private boolean needsFlush;
 
+    private boolean outboundClosed;
```

Comment:
I guess we can't use [engine.isOutboundDone()](http://docs.oracle.com/javase/7/docs/api/javax/net/ssl/SSLEngine.html#isOutboundDone())?